### ISSUE TITLE
Support webRTC without bitcode

### ISSUE THREAD

**Issue by gsl-marouanelanouari** [snapcall/snapcall_sdk_ios#5]
Do you plane to support webRTC without bitcode in spec.dependencies

**Comment by pnoyelle** [comment]
Hello @gsl-marouanelanouari  , we normally have deployed both webrtc build with and whitout bitcode. the default is with bitcode . Just add the deps in your package manager from our own build without bitcode (in git tag from github https://github.com/snapcall/WebRTC_IOS/tree/1.6.0 ). 
Ask for anymore issue or precision .

**Comment by gsl-marouanelanouari** [comment]
Yes I'm using the podfile not package manager and in the `Snapcall_Framework.podspec` the `spec.dependency 'WebRTC_IOS' , '1.6.0-bitcode'` so i can't use version 1.6.0 without bitcode.
